tests/integration/targets/lxd_storage_pool_info/tasks/main.yml

```diff
@@ -0,0 +1,69 @@
+####################################################################
+# WARNING: These are designed specifically for Ansible tests       #
+# and should not be used as examples of how to write Ansible roles #
+####################################################################
+
+# Copyright (c) Ansible project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- name: Get information about all storage pools
+  community.general.lxd_storage_pool_info:
+  register: pool_info_all
+
+- name: Verify storage pools information was retrieved
+  assert:
+    that:
+      - pool_info_all is not changed
```

Comment:
Testing for change is kinda redundant - we know it never will - it is not wrong though 😄